Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md

```diff
@@ -0,0 +1,72 @@
+---
+versionFrom: 8.0.0
+---
+
+## Non-synchronised/replicated file system
+
+If the file system on your servers isn't performing any file replication then no _Umbraco_ configuration file changes are necessary.
+
+Depending on the configuration and performance of the local storage you might need to consider Examine Directory Factory Options and the Umbraco temporary storage location. 
+
+## Synchronised/replicated or shared file system
+
+If the file system on your servers is performing file replication then the Umbraco temporary folder must be excluded from replication.
```

Comment:
What is the 'Umbraco temporary folder' that you would like them to exclude?

app_data?

Reply:
think the folders to exclude are listed further down. so can link to that section?